flow_action_components/GetFieldSetValues/force-app/main/default/classes/GetFieldSetValues.cls

```diff
@@ -0,0 +1,66 @@
+/**
+ * @description       : Pairs with ExecuteSOQL or RecordDetailFSC (or other actions) and returns a Text Collection or Comma-Separated String of fields contained in an object's Field Set. Useful
+ * For abstracting values out of the Flow so that it can be extended with minimal edits to the Flow.
+ * @author            : Adam White
+ * @last modified on  : 11-02-2020
+ * @last modified by  : Adam White
+ * Modifications Log 
+ * Ver   Date         Author                               						  Modification
+ * 1.0   11-02-2020   Adam White, https://www.linkedin.com/in/declarativeninja/   Initial Version
+**/
+
+public with sharing class GetFieldSetValues {
+    @InvocableMethod
+    public static List <Results> getFieldSetVals(List<Requests> requestList) {
```

Comment:
I've recently started using 'execute' as the name of this method, in each invocable action I build. It helps clarify that this method name isn't really used explicitly....